### PR TITLE
Add postgis image

### LINKED ISSUE #137: PostGIS images
We have a client project that use [PostGIS](https://registry.hub.docker.com/r/postgis/postgis/).
The official images are not compatible with Tugboat due to the use of `VOLUME`.

I will provide a PR to add this image and let's discuss if this should be here or if I need to maintain a community image.

### PR DESCRIPTION
**Pull request by gido**
This PR fix  #137

This add PostGIS image support to Tugboat and this is based on the PostgreSQL image.

### CODE REVIEW

File: services/postgis/manifest [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function getTags() {
+  ((newer_than=365*12*60*60))
+  curl -Ls https://registry.hub.docker.com/v2/repositories/postgis/postgis/tags?page_size=1024 |
+    jq -r '[.results[] | select((.images | length) > 0 and (.last_updated | sub("\\.[0-9]+Z"; "Z") | fromdate) > (now - '"$newer_than"')) | {name: .name, digest: .images[0].digest}] | group_by(.digest) | map([.[].name] | join(",")) | .[]' |
+    grep -v -e beta -e rc -e alpine
```
[the remark this answers is not in the document]
Comment (q0rban) [reply services/postgis/manifest:5]: This looks great, nice work!

**Review by q0rban — APPROVED** [review]

**Review by q0rban — CHANGES_REQUESTED** [review]
let's call this just `postgis`

**Comment by q0rban** [comment]
closing in favor of #139 as I couldn't figure out how to get the GHA workflow to run here.

File: services/postgis/manifest [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+}
+NAME="tugboat-postgis"
```
[the remark this answers is not in the document]
Comment (gido) [reply services/postgis/manifest:7]: Much better! I added the prefix to publish the image on my own docker orgs to test it and forget to remove it.

**Pull request closed without merging**